documentation/en/faqs.md

```diff
@@ -0,0 +1,132 @@
+
+#### Where can I get the latest version of Lotus?
+
+Download the binary tagged as the `Latest Release` from the
+ [Lotus Github repo](https://github.com/filecoin-project/lotus/releases).
+ 
+#### What operating systems can Lotus run on?
+
+Lotus can build and run on most Linux and MacOS systems with at least 
+8GB of RAM. Windows is not yet supported.
+
+#### How can I update to the latest version of Lotus?
+
+To update Lotus, follow the instructions [here](https://lotu.sh/en+updating-lotus).
+
+#### How do I prepare a fresh installation of Lotus?
+
+Stop the Lotus daemon, and delete all related files, including sealed and chain data by 
+running `rm ~/.lotus ~/.lotusstorage`.
+
+Then, install Lotus afresh by following the instructions 
+found [here](https://docs.lotu.sh/en+getting-started).
+
+## Interacting with a Lotus node
+
+#### How can I communicate with a Lotus node?
+
+Lotus nodes have a command-line interface, as well as a JSON-RPC API.
+
+#### What are the commands I can send using the command-line interface? 
+
+The command-line interface is self-documenting, try running `lotus --help` from the `lotus` home 
+directory for more.
+
+#### How can I send a request over the JSON-RPC API?
+
+Information on how to send a `cURL` request to the JSON-RPC API can be found
+[here](https://lotu.sh/en+api). A JavaScript client is under development.
+
+#### What are the requests I can send over the JSON-RPC API?
+
+Please have a look at the 
+[source code](https://github.com/filecoin-project/lotus/blob/master/api/api_common.go) 
+for a list of methods supported by the JSON-RPC API.
+## The Test Network
+
+#### What is the test network?
+
+The test network, sometimes called Testnet, is a live network of Lotus nodes run by the 
+community. It has 2 PiB of storage (and growing!) dedicated to it.
+
+#### Is FIL on the test network worth anything?
+
+Nothing at all! Real-world incentives may be provided in a future phase of the test network, but this is 
+yet to be confirmed.
+
+#### Will there be future phases of the test network?
+
+Yes, there will be at least one more phase of the test network. We plan on introducing interoperable
+[go-filecoin nodes](https://github.com/filecoin-project/go-filecoin#filecoin-go-filecoin)
+in a future phase.
+
+#### How can I see the status of the test network?
+
+The [dashboard](https://stats.testnet.filecoin.io/) displays the status of the network as 
+well as a ton
+of other metrics you might find interesting.
+
+## Mining with a Lotus node on the test network
+
+#### How do I get started mining with Lotus?
+
+Follow the instructions found [here](https://lotu.sh/en+mining).
+
+#### What are the minimum hardware requirements?
+
+An example test configuration, and minimum hardware requirements can be found 
+[here](https://lotu.sh/en+hardware-mining). 
+
+Note that these might NOT be the minimum requirements for mining on the real main network.
+
+#### What are some GPUs that work well?
```

Comment:
**Suggestion** I would change the title of this to: **What are some GPUs that the team has tested?** We don't want to give guarantees.